### DISCUSSION TITLE
로딩에 실패했습니다. 어떻게 처리하실건가요?

### DISCUSSION THREAD

**Discussion by giscus** [teamGrowing/team-blog#11] [Comments]
# 로딩에 실패했습니다. 어떻게 처리하실건가요?

비동기 처리를 ‘잘’ 해보자. 비동기 처리는 프론트엔드 개발에서 피할 수 없는 중요한 부분이에요. 특히 사용자 경험(UX)에 큰 영향을 미치는데요, 사용자에게 끊김 없는 경험을 제공하기 위해서는 로딩, 성공, 상태 등 비동기 작업의 상태를 적절하게 관리하고 표현하는 것이 필수적이죠. 이번 글에서는 그로잉 팀의 비동기 처리와 사용자 경험(UX) 향상 전략에…

https://teamgrowing.github.io/team-blog/asynchronous-processing/

**Comment by Joong-Sunny** [comment]
좋은 구조 잘 보고 갑니다..!

**Reply by Joong-Sunny** [reply]
공부겸 코드도 확인해봤는데, Axios에서 제공하는 타입가드 쓰면 더 편하지 않을까 하는 생각도 드네요

```diff
- import { AxiosError } from 'axios';
-  const isAxiosError = (err: Error) => {
-    return err instanceof AxiosError;
-  };

-  if (!isAxiosError(error)) {
-    throw error;
-  }

+ import Axios from 'axios';
+  if (!Axios.isAxiosError(error)) {
+    throw error;
+  }

```
